src/lib/y2security/clients/security_policy_proposal.rb

```diff
@@ -393,6 +437,12 @@ def to_html
               link:   toggle_link
             )
           end
+
+          if policy_enabled
```

Comment:
NP: Why cannot we add these sections in the if above?

Reply:
Yes, you are right. I changed this code a few times, but the final result does not make sense :-) Fixed.